medium-tests/src/test/java/mediumtest/ConnectionSuggestionMediumTests.java

```diff
@@ -383,4 +409,14 @@ void should_suggest_sonarcloud_connection_when_pascal_case(@TempDir Path tmp) th
     assertThat(connectionSuggestion.get(CONFIG_SCOPE_ID).get(0).isFromSharedConfiguration()).isTrue();
   }
 
+  private static Stream<Arguments> emptyBindingSuggestionsTestValueProvider() {
+    return Stream.of(
+      Arguments.of("", "", ""),
+      Arguments.of("", "foo", ""),
+      Arguments.of("", "", "foo"),
+      Arguments.of("key", "foo", ""),
```

Comment:
As discussed on our call, it would be good to have a test which asserts `fakeClient.getSuggestionsByConfigScope()` is non-empty for this case (which has a projectKey and SQ URL) +same for the two valid cased bellow